src/utils/get-html-from-native-paste.js

```diff
@@ -0,0 +1,49 @@
+import { findDOMNode } from 'react-dom'
+
+/**
+ * Get clipboard html data by capturing the html inserted by the browser's native paste action.
+ *
+ * To make this work, `preventDefault()` may not be called on the `onPaste` event.
+ * If pasted html can be retreived, it is added to the given `data` object, setting the `type` to `html`.
+ * As this method is asynchronous, a callback is needed to return the `data` object.
+ *
+ * Solution adapted from http://stackoverflow.com/a/6804718).
+ *
+ * @param {React.Component} component
+ * @param {Object} data
+ * @param {Function} callback
+ */
+
+function getHtmlFromNativePaste(component, data, callback) {
+  const contentNode = findDOMNode(component)
+
+  const clipboardNode = contentNode.cloneNode()
```

Comment:
Can you change these variable names to `contentNode -> el` and `clipboardNode -> clone`?